play/plugin/build.gradle.kts

```diff
@@ -51,6 +51,11 @@ afterEvaluate {
 tasks.withType<Test> {
     inputs.files(fileTree("src/test/fixtures"))
 
+    // AGP 8 requires JDK 17 and we want to to be compatible with previous JDKs
+    javaLauncher.set(javaToolchains.launcherFor {
+        languageVersion.set(JavaLanguageVersion.of(17))
+    })
```

Comment:
Even AGP 8 can be consumed by JDK11, it has a hard check on runtime to require JDK17. I've workaround this by changing the toolchain on the `test` task. Bumping the whole build directly to JDK17 may affect compatibility with the consumers of the plugins.